packages/communication/src/abi.ts

```diff
@@ -0,0 +1,35 @@
+import { AbiInput, AbiItem } from 'web3-utils'
+import { EventLog } from './commons'
```

Comment:
I'm going to assume that there aren't significant code changes and just focus on structure. Please lmk if there's any files you want me to check for code.